src/dashboard/src/components/administration/tests/test_dip_upload_configs.py

```diff
@@ -60,15 +60,23 @@ def test_post_missing_fields(self):
         self.assertFalse(form.is_valid())
         self.assertTrue(form.errors)
 
-        self.assertFormError(response, "form", "user", "This field is required.")
-        self.assertFormError(response, "form", "xlink_show", "This field is required.")
         self.assertFormError(
-            response, "form", "xlink_actuate", "This field is required."
+            response.context["form"], "user", "This field is required."
```

Comment:
This change is documented in https://docs.djangoproject.com/en/4.2/releases/4.1/#id2 under:
>Passing a response object and a form/formset name to SimpleTestCase.assertFormError() and assertFormsetError() is deprecated

Reply:
Got it. Thank you !